test/test.js

```diff
@@ -0,0 +1,49 @@
+﻿const mocha = require('mocha');
+const chai = require('chai');
+const program = require('../words');
+const assert = chai.assert;
+chai.config.truncateThreshold = 0;
+describe('Test words function', () => {
+    it('should handle specified input', () => {
+        let testcase = program.words('olly olly in come free');
+        assert.deepEqual(testcase, { olly: 2, in: 1, come: 1, free: 1 });
+    });
+    it('should handle more complex input', () => {
+        let testcase = program.words('olly olly has come in the free has the free olly');
+        assert.deepEqual(testcase, { 'olly': 3, 'has': 2, 'come': 1, 'in': 1, 'the': 2, 'free': 2 });
+    });
+    it('should return an object', () => {
+        let testCase = typeof program.words('olly olly has come in the free has the free olly');
+        assert.deepEqual(testCase, 'object');
+    });
+    it('should handle multiple spaces', () => {
```

Comment:
The `it` block here could better be phrased as "It should strip off extra unwanted spaces in any input sentence/parameter"

Reply:
Thanks a lot. Would rephrase immediately